config/locales/en.yml

```diff
@@ -402,10 +402,10 @@ en:
           title: Get benefits and support
           href: /check-benefits-financial-support
           image_src: homepage/cost-of-living.jpg
-        - text: Find out about the UK's response to the Russian government's invasion of Ukraine and how you can help.
-          title: War in Ukraine
-          href: https://ukstandswithukraine.campaign.gov.uk/
-          image_src: homepage/ukraine-promo.jpg
+        - text: High temperatures can be harmful to your health. Learn how to stay safe in hot weather.
+          title: Staying safe in hot weather
+          href: https://www.gov.uk/government/publications/heatwave-plan-for-england/beat-the-heat-staying-safe-in-hot-weather
```

Comment:
Minor - convention is to have just a path for internal links (keeps it environment agnostic)

Reply:
Thanks, addressed in 41d5a70